Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -33,12 +31,9 @@
 		3BA56FF82A9FDB5A0081D14F /* CardPaymentOrderCompletionView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3BA56FF72A9FDB5A0081D14F /* CardPaymentOrderCompletionView.swift */; };
 		3BA56FFA2A9FE4180081D14F /* CardOrderCompletionResultView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3BA56FF92A9FE4180081D14F /* CardOrderCompletionResultView.swift */; };
 		3BA56FFC2A9FEFE90081D14F /* PayPalWebViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3BA56FFB2A9FEFE90081D14F /* PayPalWebViewModel.swift */; };
```

Comment:
Could be for another PR, but could we rename `PayPalViewModel.swift` --> `PayPalNativeViewModel.swift` to follow the same convention as the Web one? Or is there a plan to further consolidate the 2?

Reply:
Yeah, I was leaving that for a follow up PR when we do those files as a whole. I _think_ they will need to be separate but agree that one should be renamed to PayPalNative. 

Right now we also have a separate directory for the Views vs the ViewModels. Do we like that pattern or should we move each features ViewModel into it's top level directory? It could be nice for merchants to just look in 1 place for everything related to a feature vs traversing multiple directories.

Reply:
I like that idea - breaking it up by feature instead. Things are a little hard to find ATM